src/controller.cpp

```diff
@@ -49,25 +50,58 @@ void Controller::init()
 {
   prev_time_us_ = 0;
 
-  float max = RF_.params_.get_param_float(PARAM_MAX_COMMAND);
-  float min = -max;
+  // Calculate the max thrust to convert from throttle setting to thrust
+  calculate_max_thrust();
+
+  max_throttle_ = RF_.params_.get_param_float(PARAM_RC_MAX_THROTTLE);
+
+  // Don't saturate the torque values
+  float max_torque = INFINITY;
+  float min = -max_torque;
```

Comment:
Should we rename this to be min_torque, since max is being renamed to max_torque? That would make it more clear that they're related.